### ISSUE TITLE
gr.Image appears different when running local vs on Spaces

### ISSUE THREAD

**Issue by dwipper** [gradio-app/gradio#6228]
### Describe the bug

The following code displays differently when running local vs running on Spaces:

with (gr.Blocks(CIMStheme) as iface):
    with gr.Row():
        with gr.Column():
            gr.Image(label="Logo",value="CIMS Logo Purple.png",height=50,show_download_button=False,interactive=False,show_label=False,elem_id="logo",container=False)

On the local, the size of the logo seems to follow the height=50 param. It appears the Image component resizes the .png. On Spaces, the component doesn't center on the page, and is cut off by 50%. If I change the height to 100 on the Spaces, it displays the full .png file, although the size is much bigger than when running local.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

with (gr.Blocks(CIMStheme) as iface):
    with gr.Row():
        with gr.Column():
            gr.Image(label="Logo",value="CIMS Logo Purple.png",height=50,show_download_button=False,interactive=False,show_label=False,elem_id="logo",container=False)

### Screenshot

![image](https://github.com/gradio-app/gradio/assets/140648761/65819b2c-932b-45dc-b5b8-4315b268d850)
![image](https://github.com/gradio-app/gradio/assets/140648761/187a08b2-98d8-4db5-9f2d-296962e20983)
![image](https://github.com/gradio-app/gradio/assets/140648761/2c3d6ce7-a75f-465b-a48b-3db9f82f1778)

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Darwin
gradio version: 4.0.2
gradio_client version: 0.7.0

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 22.1.0
altair: 5.1.2
fastapi: 0.104.0
ffmpy: 0.3.1
gradio-client==0.7.0 is not installed.
httpx: 0.25.0
huggingface-hub: 0.15.1
importlib-resources: 6.1.0
jinja2: 3.1.2
markupsafe: 2.1.1
matplotlib: 3.7.2
numpy: 1.24.3
orjson: 3.9.9
packaging: 23.1
pandas: 2.0.3
pillow: 9.4.0
pydantic: 2.4.2
pydub: 0.25.1
python-multipart: 0.0.6
pyyaml: 6.0
requests: 2.31.0
semantic-version: 2.10.0
tomlkit==0.12.0 is not installed.
typer: 0.9.0
typing-extensions: 4.8.0
uvicorn: 0.23.2
websockets: 11.0.3
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2023.4.0
httpx: 0.25.0
huggingface-hub: 0.15.1
packaging: 23.1
requests: 2.31.0
typing-extensions: 4.8.0
websockets: 11.0.3
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
What's the version of Gradio running on Spaces? Can you link to the Spaces?

**Comment by dwipper** [comment]
Hi @abidlabs . This is the space: https://huggingface.co/spaces/dwipper/NILI-Mobile

It's running https://gradio-builds.s3.amazonaws.com/66fa843699f63e72b2ed2ddf94370350bebf9769/gradio-3.50.2-py3-none-any.whl since the auth login page styling/function is broken again in 4.0.2: https://github.com/gradio-app/gradio/issues/5932

**Comment by dwipper** [comment]
I also have a bigger issue that I can't seem to figure out. I have a public space running the auth login, that points to the above private space app. The issue is that I need to pass the username from the public to the private space, since it doesn't seem like any of the global variables I set in the login get passed over the app running in the private space? Is there some trick to getting the username from the login in the private space app?

**Comment by dwipper** [comment]
This appears to be fixed in 4.1.1
